web/targetApp/views.py

```
@@ -330,12 +330,13 @@ def delete_targets(request, slug):
     if request.method == "POST":
         list_of_domains = []
         for key, value in request.POST.items():
-            if key not in ("list_target_table_length", "csrfmiddlewaretoken"):
+            if key != "list_target_table_length" and key != "csrfmiddlewaretoken":
+                list_of_domains.append(value)
                 Domain.objects.filter(id=value).delete()
```

Ruff

`Domain` may be undefined, or defined from star imports

(F405)